src/api/spec/features/beta/webui/users/notifications_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'browser_helper'
+
+RSpec.describe 'User notifications', type: :feature, js: true, beta: true do
```

Comment:
As @saraycp pointed out, after moving the test into `spec/features/beta/`, you don't need to define the `beta: true` metadata.

Reply:
True, I'll remove it

Reply:
Done